templates/case-studies/templates/_macros-highlights.jinja

```diff
@@ -0,0 +1,23 @@
+# Params
+# list_items: List of highlight items (requires 3-4 items)
+
+{%- macro highlights(list_items) -%}
+  <section class="p-strip">
+    <div class="row">
+      <p class="p-text--small-caps">Highlights</p>
+      <hr />
+    </div>
+    {% if list_items | length == 3 %}
+      {% set col_size = "col-4" %}
+    {% elif list_items | length == 4 %}
+      {% set col_size = "col-3" %}
+    {% endif %}
+    <div class="row">
+      {% for item in list_items %}
```

Comment:
Your params documentation suggests that this macro supports `list_items` length of 3 or 4. What should happen if a list of size outside that range is passed in?

If this is truly to be limited to 4 items, [Tiered List](https://github.com/canonical/vanilla-framework/blob/3885de3af2c1393b5a51302142b3a17dc6e8dd4d/templates/_macros/tiered-list.jinja#L59) may be a good example (it limits input to 25 list items by not displaying anything past the 25th element).

Reply:
We are limiting content writers to pass 3-4 `list_items` only. If any more items are passed, it should not be rendered. Thanks for linking the tiered list, it is quite similar to what the highlights section is trying to achieve. I will use it as reference.